app/assets/stylesheets/spree/frontend/components/layout/_header.scss

```diff
@@ -16,7 +16,46 @@
     &__content {
       align-items: center;
       display: flex;
+      flex-wrap: wrap;
+      justify-content: space-between;
       padding: 2rem 3rem;
+
+      @include desktop {
+        flex-wrap: nowrap;
+        justify-content: flex-end;
+      }
+
+      .logo {
```

Comment:
We should move this styles to `_logo.scss`. Is there a reason why they are here?

Reply:
I think that I get a similar comment before, regarding where to put style rules that belongs to the aggregation of components. For example, having the `order` inside the single component does not make sense to me because the same component could have a different order if it's used in another aggregation. In this case, the header aggregates a logo, search-bar, auth and cart links and sort them according to mobile/desktop layout.